shared/teams/team/index.js

```diff
@@ -388,18 +388,17 @@ const CustomComponent = ({onOpenFolder, onManageChat, onShowMenu, canManageChat}
       canManageChat && (
         <Icon
           onClick={onManageChat}
-          type="iconfont-chat"
-          style={{fontSize: isMobile ? 20 : 16, marginRight: globalMargins.tiny}}
-        />
-      )}
-    {!isMobile &&
-      canManageChat && (
-        <Icon
-          onClick={onOpenFolder}
-          type="iconfont-folder-private"
           style={{fontSize: isMobile ? 20 : 16, marginRight: globalMargins.tiny}}
+          type="iconfont-chat"
         />
       )}
+    {!isMobile && (
```

Comment:
I think the folder icon needs to disappear for users not in the team, because clicking it always gives `Uncaught (in promise) Error: Unable to open/stat file: /keybase/team/...` and does nothing.